bundle-hack/container_digest.sh

```diff
@@ -0,0 +1,9 @@
+# Do not remove comment lines, they are there to reduce conflicts
+# Operator
+export OPERATOR_IMAGE_PULLSPEC='quay.io/redhat-user-workloads/external-dns-operator-tenant/ext-dns-optr-1-2-rhel-8/external-dns-operator-container-ext-dns-optr-1-2-rhel-8@sha256:123'
+# Controller
+export OPERAND_IMAGE_PULLSPEC='quay.io/redhat-user-workloads/external-dns-operator-tenant/ext-dns-optr-1-2-rhel-8/external-dns-container-ext-dns-optr-1-2-rhel-8@sha256:123'
```

Comment:
Similar to the above. We have a valid operand image which we could use already and have the bundle pipeline successful on merge.
```suggestion
export OPERAND_IMAGE_PULLSPEC='quay.io/redhat-user-workloads/external-dns-operator-tenant/ext-dns-optr-1-2-rhel-8/external-dns-container-ext-dns-optr-1-2-rhel-8@sha256:9c5b04c799dc0333226ec691b8d2a2950231d52aebca307d13f8b01d38881358'
```

Reply:
Done.